### PR TITLE
feat: implemented penality to unsolicited peers meesages

### PR DESCRIPTION
**Pull request by twjang**
Hello! I Implemented the feature described in https://github.com/witnet/witnet-rust/issues/739.

### CODE REVIEW

File: node/src/actors/session/handlers.rs [hunk 1]
```diff
@@ -391,6 +391,8 @@ impl Handler<SendGetPeers> for Session {
 
     fn handle(&mut self, _msg: SendGetPeers, _: &mut Context<Self>) {
         log::trace!("Sending GetPeers message to peer at {:?}", self.remote_addr);
+
+        self.expected_peers_msg += 1;
```
Comment (tmpolaczyk) [inline node/src/actors/session/handlers.rs:395]:
```suggestion
        self.expected_peers_msg = self.expected_peers_msg.saturating_add(1);
```

File: node/src/actors/session/mod.rs [hunk 2]
```diff
@@ -84,6 +84,9 @@ pub struct Session {
 
     /// Reference to config
     config: Arc<Config>,
+
+    /// Expected number of "peers" message from this peer
+    expected_peers_msg: i32,
```
Comment (tmpolaczyk) [inline node/src/actors/session/mod.rs:89]: I think this cannot be negative, and u8 should be enough

```suggestion
    expected_peers_msg: u8,
```

**Comment by tmpolaczyk** [comment]
Hi, thanks for the PR! I added a few small comments but overall looks good. Sorry for the delay.

**Comment by twjang** [comment]
Hi! I squashed all changes into one commit, and applied improvements made in the master branch. 

But "build_ubuntu" github action seems to be broken somehow..

**Review by tmpolaczyk — APPROVED** [review]

**Pull request merged**